internal/services/engines/javascript/rules.go

```diff
@@ -330,7 +330,7 @@ func NewHavingAPermissiveCrossOriginResourceSharingPolicy() text.TextRule {
 		},
 		Type: text.Regular,
 		Expressions: []*regexp.Regexp{
-			regexp.MustCompile(`Access-Control-Allow-Origin\s*:\s*['|"]\*['|"]`),
+			regexp.MustCompile(`Access-Control-Allow-Origin["|'|:][\s|,][\s|"|']['|"|\*]['|"|\*]\D`),
```

Comment:
I changed the rule to get another very common scenario in javascript.

`res.header("Access-Control-Allow-Origin", "*");`

![image](https://user-images.githubusercontent.com/69604366/143292533-bfa994a4-741b-40f2-b844-3e4b4c0ad53f.png)